.github/workflows/nightly.yml

```diff
@@ -0,0 +1,189 @@
+name: Nightly testing workflows for colcon-bundle
+on:
+  # Build at minute 43 hour 17 (9:43am PST)
+  # Minutes was chosen randomly
+  schedule:
+    - cron: '43 17 * * *'
+
+jobs:
+  tox_unittest:
+    runs-on: ubuntu-18.04
+    env:
+      TOXENV: unittest
+    steps:
+    - name: Checkout sources
+      uses: actions/checkout@v2
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: ${{ secrets.AWS_REGION }}
+    - name: Setup Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.5
+    - name: Install dependencies
+      run: |
+        sudo apt update && sudo apt install -y enchant
+        pip install -U tox setuptools==44.0.0
+    - name: Run tests
+      run: tox
+    - name: Log Test Failure
+      uses: ros-tooling/action-cloudwatch-metrics@0.0.1
+      with:
+        status: 'failure'
+      if: failure() && github.event_name == 'schedule'
+    - name: Log Test Success
+      uses: ros-tooling/action-cloudwatch-metrics@0.0.1
+      with:
+        status: 'success'
+      if: success() && github.event_name == 'schedule'
```

Comment:
Do we still need to check `github.event_name == 'schedule'` here?

Reply:
I think it's a good sanity check.
Maybe down the line we might add another event and forget about this.

Reply:
if this one is only run on schedule, perhaps it should be in its own workflow file that is explicitly for scheduled events?

Reply:
> if this one is only run on schedule, perhaps it should be in its own workflow file that is explicitly for scheduled events?

I agree with that. It'd be a good idea to separate cron jobs from ad-hoc workflows like PRs and pushes.

Reply:
removed from `if-condition`